mj_envs/tests/envs/biomechanics/test_biomechanics.py

```diff
@@ -0,0 +1,36 @@
+import gym
+import mj_envs.envs.biomechanics
+import pickle
+import pytest
```

Comment:
Maybe we can harmonize this test with more tests that are available here -- https://github.com/vikashplus/mj_envs/blob/dambrl/mj_envs/tests/test_envs.py